### PR TITLE
readme: add space after starship symbol

### PR DESCRIPTION
**Pull request by basbebe**

### CODE REVIEW

**Comment by alexherbo2** [comment]
Can you squash the commits into a single one?

**Comment by basbebe** [comment]
sorry. sure, will do!

**Comment by basbebe** [comment]
done.

File: README.md [hunk 1]
```diff
@@ -275,6 +275,7 @@ when = 'test "$IN_KAKOUNE_CONNECT" = 1 -a -n "$KAKOUNE_SESSION" -a -n "$KAKOUNE_
 shell = ['dash']
 description = 'Indicates that the current shell is connected to a Kakoune client'
 style = 'green italic'
+format = "[$symbol ($output )]($style)"
```
Comment (alexherbo2) [inline README.md]: Prefer literal strings.

File: README.md [hunk 2]
```diff
@@ -283,6 +284,7 @@ when = 'test "$IN_KAKOUNE_CONNECT" = 1 -a -n "$KAKOUNE_SESSION" -a -z "$KAKOUNE_
 shell = ['dash']
 description = 'Indicates that the current shell is connected to a Kakoune session'
 style = 'green italic'
+format = "[$symbol ($output )]($style)"
```
Comment (alexherbo2) [inline README.md]: Same here.

**Review by alexherbo2 — CHANGES_REQUESTED** [review]

**Comment by basbebe** [comment]
like so?

**Comment by alexherbo2** [comment]
Yep perfect :ok_hand:

**Pull request merged**